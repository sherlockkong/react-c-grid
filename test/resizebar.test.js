import React from 'react';
import ResizeBar from '../src/components/ResizeBar';
import renderer from 'react-test-renderer';

it('resizebar', () => {
	const resizebar = renderer
		.create(<ResizeBar colIndex={0} column={{}} />)
		.toJSON();
	expect(resizebar).toMatchSnapshot();
});