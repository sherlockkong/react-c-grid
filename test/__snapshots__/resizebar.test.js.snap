// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resizebar 1`] = `
<div
  className="col-resize-bar"
  data-col-index="0"
  onMouseDown={[Function]}
>
  <div
    className="holder holder-left"
  />
  <div
    className="holder holder-right"
  />
</div>
`;
